### ISSUE TITLE
MessageView should have accessibleDescription(aria-describedby) prop or by default have "List with interactive items."

### ISSUE THREAD

**Issue by JJPandari** [SAP/ui5-webcomponents-react#8196]
### Is your feature request related to a problem?

Yes. Because message items in message view can have interactive content like links, a11y tester demands we give screen readers a description "List with interactive items. To move to the content press F2." But MessageView doesn't have a accessibleDescription prop.

### Describe the solution you'd like

One of the following:

1. MessageView have a prop `accessibleDescription`, of which the content is added to the `ul`
2. the generated `ul` always has a `aria-describedby="List with interactive items. To move to the content press F2."` If I understand right, UI5 does this, the generated html has this without the need to write it in app code: https://sapui5untested.int.sap.eu2.hana.ondemand.com/#/entity/sap.m.MessagePopover/sample/sap.m.sample.MessagePopover

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @JJPandari 

when the linked PR is merged and the changes are released, it will be possible setting the `accessible-description` and `accessible-description-ref` attribute on the `List` component used in the `MessageView` by setting `accessibleDescriptionRef` or `accessibleDescription`.

Also, we provide an example to show how to reuse the ui5wcr localized text.

**Comment by Lukas742** [comment]
Hi @JJPandari 

quick update. With v2.20.0 of `@ui5/webcomponents`, the `List` itself will add the description. So, after updating, no additional action is required to achieve the desired behavior. We will publish a compatible version soon.
